public/js/actions/subscriptions.js

```diff
@@ -0,0 +1,33 @@
+import $ from 'jquery';
+
+import * as actionTypes from 'constants/action-types';
+
+import * as appActions from './app';
+
+
+export function getUserSubscriptions(jquery=$) {
+  return dispatch => {
+    dispatch({
+      type: actionTypes.LOADING_USER_SUBSCRIPTIONS,
```

Comment:
this is an interesting case. It's actually not possible to put the spinner on the component alone because the data can be reloaded. When it gets reloaded, the component already exists (it's showing the old subscription data). I've seen the pattern of dispatching loading actions in other redux examples, I think it works. It's nice because the data reducer can reset itself when it gets a loading action.